generate_geofence.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: UTF-8 -*-
+import json
+import urllib2
+import csv
+import configargparse
+
+
+def main():
+
+    # ARGS
+    parser = configargparse.ArgParser(description='geoFenceGen')
+    parser.add_argument(
+        '-dir',
+        '--directory',
+        help='Directory and filename of geofence. ex: geofence/Bratislava.txt',
+        required=True)
+    parser.add_argument(
+        '-l',
+        '--location',
+        help='City/Country/area you would like a geofence for',
+        required=True)
+
+    parser.add_argument(
+        '-w',
+        '--way',
+        help=('Use if there\'s no relational polygon' +
+              '(ex: Manhattan central park'),
+        action='store_true',
+        default=False)
+    parser.add_argument(
+        '-mp',
+        '--multiPolygon',
+        help=('Use if you wish to write all polygons of an area'),
+        action='store_true',
+        default=False)
+    args = parser.parse_args()
+    loc = (args.location)
+    loc = loc.replace(" ", "+")
+    end = ("https://nominatim.openstreetmap.org/"
```

Comment:
End of what? Variable should ve called url or osm_url or something more descriptive.

Reply:
short for end*point*. can change if matters